File: spark-action/src/main/java/com/skrein/spark/sql/action/opp/Sub.scala
package com.skrein.spark.sql.action.opp

/**
 *
 *
 * @author :hujiansong  
 * @date :2019/11/12 18:05
 * @since :1.8
 *
 */
class Sub extends Supper {

  def method()={
      println(this.name)
  }




}


object App{


  def main(args: Array[String]): Unit = {
    val sub = new Sub
    sub.method()

  }
}

File: spark-action/src/main/java/com/skrein/spark/sql/action/weibo/Sub.java
package com.skrein.spark.sql.action.weibo;

/**
 * @author :hujiansong
 * @date :2019/11/12 18:04
 * @since :1.8
 */
public class Sub extends Supper {
    public void method(Supper supper){
        System.out.println(this.name);
        System.out.println(supper.name);
    }

    public static void main(String[] args) {
        new Sub().method(new Sub());
    }
}
